docs/zz_generated.seed.yaml

```diff
@@ -139,6 +139,8 @@ spec:
           # See https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#load-balancer
           # This setting defaults to true.
           manage_security_groups: true
+          # Optional: Gets mapped to "use-octavia" setting in the cloud config. This setting defaults to true.
```

Comment:
Did you update this by running `hack/update-docs.sh`? I'm asking because the validation seems to fail.

Reply:
No, I didn't. Where is this documented?
The new push includes the corrected docs.

Reply:
Good question :-), this codebase used to be closed source and we are aware we have room for improvement regarding contributors experience.  I'll open an issue to write a contributing guide ;-)

Reply:
BTW I forgot to mention, but you have also to run `hack/update-swagger.sh` ;-)

Reply:
I noticed already, see last commit.
But, `./hack/verify-api-client.sh` is also required ;-) Would be better to call `make verify` directly, but one has to know that.

Reply:
Indeed, I'll put all this in the issue as promised ;-)

Reply:
👍